get-started/06_bind_mounts.md

```diff
@@ -1,121 +1,223 @@
+[nodemon](https://npmjs.com/package/nodemon){:target="_blank" rel="noopener"
+class="_"} to watch for file changes, and then restart the application
+automatically. There are equivalent tools in most other languages and
+frameworks.
 
 ## Quick volume type comparisons
 
-Bind mounts and named volumes are the two main types of volumes that come with the Docker engine. However, additional
-volume drivers are available to support other use cases ([SFTP](https://github.com/vieux/docker-volume-sshfs){:target="_blank" rel="noopener" class="_"}, [Ceph](https://ceph.com/geen-categorie/getting-started-with-the-docker-rbd-volume-plugin/){:target="_blank" rel="noopener" class="_"}, [NetApp](https://netappdvp.readthedocs.io/en/stable/){:target="_blank" rel="noopener" class="_"}, [S3](https://github.com/elementar/docker-s3-volume){:target="_blank" rel="noopener" class="_"}, and more).
+The following table outlines the main differences between volume mounts and bind
+mounts.
 
-|   | Named Volumes | Bind Mounts |
-| - | ------------- | ----------- |
-| Host Location | Docker chooses | You control |
-| Mount Example (using `-v`) | my-volume:/usr/local/data | /path/to/data:/usr/local/data |
-| Populates new volume with container contents | Yes | No |
-| Supports Volume Drivers | Yes | No |
+|                                              | Named volumes                                      | Bind mounts                                          |
+| -------------------------------------------- | -------------------------------------------------- | ---------------------------------------------------- |
+| Host location                                | Docker chooses                                     | You decide                                           |
+| Mount example (using `--mount`)              | `type=volume,src=my-volume,target=/usr/local/data` | `type=bind,src=/path/to/data,target=/usr/local/data` |
+| Populates new volume with container contents | Yes                                                | No                                                   |
+| Supports Volume Drivers                      | Yes                                                | No                                                   |
 
-## Start a dev-mode container
+## Trying out bind mounts
 
-To run our container to support a development workflow, we will do the following:
+Before looking at how we can use bind mounts for developing our application,
+let's run a quick experiment to get a practical understanding of how bind mounts
+work.
 
-- Mount our source code into the container
-- Install all dependencies, including the "dev" dependencies
-- Start nodemon to watch for filesystem changes
+1. Open a terminal and make sure your current working directory is in the `app`
+   directory of the getting started repository.
 
-So, let's do it!
+2. Run the following command to start `bash` in an `ubuntu` container with a
```

Comment:
I think pwd won't work in Command Prompt. For Windows users, I suggest to mention to run this in PowerShell.
Also, small consistency nit, we use `$PWD` here, but `"$(pwd)"` in some other commands below.

Reply:
thanks, good eye. I'll make these consistent. Does this commmand work as-is in PS1? `$ docker run -it --mount type=bind,src=$PWD,target=/src ubuntu bash`

Reply:
There is a `pwd` program in PowerShell.
`$ docker run -it --mount type=bind,src="$(pwd)",target=/src ubuntu bash` works in PowerShell.

Reply:
grand, then this procedure should work with powershell!